### PR TITLE
Fixed view talent command to accept multi word names

### PR DESCRIPTION
**Pull request by JeremyTHolland**
The fix is simple. Instead of only taking the second word of the input as the talent name which breaks on multi-word names, the code takes the entire string after the view command as the input to act upon.

### CODE REVIEW

File: src/WarriorRogueMage/CharacterCreation/PickTalentsState.cs [hunk 1]
```diff
@@ -84,7 +85,7 @@ private void ViewTalentDescription(string talent)
         {
             string talentToView = talent.Replace("view ", string.Empty);
 
-            Talent foundTalent = this.talents.Find(s => s.Name.Equals(talentToView, StringComparison.CurrentCultureIgnoreCase));
+            Talent foundTalent = this.talents.Find(s => s.Name.Contains(talentToView, StringComparison.CurrentCultureIgnoreCase));
```
Comment (DavidRieman) [inline src/WarriorRogueMage/CharacterCreation/PickTalentsState.cs:88]: Should we StartsWith OR Contains to prioritize talents starting with it?

Comment (JeremyTHolland) [reply src/WarriorRogueMage/CharacterCreation/PickTalentsState.cs:88]: Like this?
`Talent foundTalent = this.talents.Find(s => s.Name.StartsWith(talentToView, StringComparison.CurrentCultureIgnoreCase) || s.Name.Contains(talentToView, StringComparison.CurrentCultureIgnoreCase));`

Comment (DavidRieman) [reply src/WarriorRogueMage/CharacterCreation/PickTalentsState.cs:88]: Yep. The total time spent processing this is inconsequential. The order here means if user just gives like two characters it should result in finding the one they probably meant over one with those teo chars in the middle.

Comment (JeremyTHolland) [reply src/WarriorRogueMage/CharacterCreation/PickTalentsState.cs:88]: OK, I'll make the update. Thanks!

**Review by DavidRieman — APPROVED** [review]

**Pull request merged**